requirements/base/base.in

```diff
@@ -1,10 +1,12 @@
 # base requirements.in
-django>2.2.16,<3.0
+django==3.2
 celery
 census
 us
 dealer
-django-cache-machine
+# Use forked version until package is updated
+# django-cache-machine
+git+https://github.com/Afani97/django-3.2-cache-machine@master#egg=django-3.2-cache-machine
```

Comment:
I'm pretty sure @tobiasmcnulty is a maintainer on django-cache-machine:

<img width="508" alt="image" src="https://user-images.githubusercontent.com/129679/177358454-c6b5fd23-62f6-4903-afdb-d963b8f9ae47.png">

Maybe we can ping him directly or get you a commit bit on it? 😄 

Alternatively, I wonder if it would be valuable to consider another cache package, but I don't know how easy it would be to switch.

Reply:
I have an open PR for a Django upgrade for that package which needs a review and if it's merged then that's great! Otherwise I wasn't able to find a comparable package yet.